pkgs/development/python-modules/duckduckgo-search/default.nix

```diff
@@ -13,24 +13,24 @@
 
 buildPythonPackage rec {
   pname = "duckduckgo-search";
-  version = "6.3.2";
+  version = "6.3.4";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "deedy5";
     repo = "duckduckgo_search";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-5AuPAv78ePrnCr5L4CfIu/fq7Ha19zC78zg8JLu3U2A=";
+    rev = "v${version}";
+    hash = "sha256-NvFoiyoXeNOrynGN+VHVfIA3+D9zfAWFeWEVU8/TkZQ=";
   };
 
   build-system = [ setuptools ];
 
   dependencies = [
     click
     primp
-  ];
+  ] ++ optional-dependencies.lxml;
```

Comment:
Why this?

Reply:
Because this is an optional dependency upstream. https://github.com/deedy5/duckduckgo_search/blob/9b73864c0e7d8145d99f46d4f4b6e93a069573b9/pyproject.toml#L47

I hadn't included this line when I added it and just saw it, because without it, it will not works

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#how-to-add-optional-dependencies-python-optional-dependencies